src/components/ButtonPanel.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div id="btnPanel">
+      <div className="rows">
+        <Button name="AC" />
```

Comment:
In your `Button.js` component, the name of your `prop` is `btnName` but you're passing `name` into it here. You must have noticed your buttons are not displaying in the browser and that's because you have not truly passed in the `btnName` prop you specified in the `Button` component. Change all instances `name` here to `btnName` or change `btnName` in the `Button` component to `name`.

Reply:
Solved